### PR TITLE
fix: reduce PIN attempts in Send and ChangePIN flows

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

 - extract all complex PIN logic to the hook
 - use this hook in all 3 flows: General auth, Change PIN, Send flows
 - disable annoying eslint rule, we don't need it, we have prettier

### Type of change

Bug fix

### Tests

Detox test

### CODE REVIEW

**Review by pwltr — APPROVED** [review]
tested, LGTM

File: src/components/PinPad.tsx [hunk 1]
```diff
@@ -302,16 +178,7 @@ const styles = StyleSheet.create({
...
-		justifyContent: 'center',
-		marginBottom: 48,
-	},
-	dot: {
-		width: 20,
-		height: 20,
-		borderRadius: 10,
-		marginHorizontal: 12,
-		borderWidth: 1,
+		marginBottom: 'auto',
```
Comment (pwltr) [inline src/components/PinPad.tsx]: Small spacing issue here between the biometry button and the attempts text 
image

Comment (pwltr) [reply src/components/PinPad.tsx]: Also here (horizontal padding)
image

Comment (limpbrains) [reply src/components/PinPad.tsx]: fixed this

**Pull request merged**